map/place_page_info.hpp

```diff
@@ -48,6 +48,9 @@ class Info : public osm::MapObject
   /// TODO: Support all possible Internet types in UI. @See MapObject::GetInternet().
   bool HasWifi() const;
 
+  /// Returns true, if the current time is within the working range of the banner.
+  bool HasBanner() const;
```

Comment:
Please, move this method to corresponding getters and surround them by blank lines.